src/org/usfirst/frc4904/robot/commands/Climb.java

```diff
@@ -23,7 +25,7 @@ protected void execute() {
 		set(Math.max(0, axis.get())); // No, really, always feed positive values
 										// to the climber motor
 		if (axis.get() >= Climb.MINIMUM_AXIS_VALUE_TO_FLIP_GEARIO_RAMP && RobotMap.Component.gearIO.isNotOverridden()) {
```

Comment:
While the command will interrupt previous copies, I think it would be better to have it only started if it is not already running (to try to minimize weird scheduler cases that we have to deal with). Same applies below.